lib/CMakeLists.txt

```diff
@@ -298,9 +224,10 @@ if(DYNAMIC_LIB)
   install(
     FILES "${CMAKE_CURRENT_BINARY_DIR}/${PROJECT_NAME}-config.cmake"
           "${CMAKE_CURRENT_BINARY_DIR}/${PROJECT_NAME}-config-version.cmake"
+          "${CMAKE_CURRENT_SOURCE_DIR}/cmake/${PROJECT_NAME}-conan.cmake"
     DESTINATION lib/cmake/scipp
```

Comment:
Should this also use `PROJECT_NAME`? (Same below)

Reply:
It is? What are you referring to?

Reply:
Oh, you don't mean the changed line, but the line below?

Reply:
Yes